server/map_rpc_server_test.go.old

```diff
@@ -0,0 +1,197 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Is this file supposed to be updated or removed?

Reply:
It's not supposed to have been put in a commit, that's for sure!